src/content/nl/2020/markup.md

```diff
@@ -177,16 +171,13 @@ In totaal bevat ongeveer 2% van de pagina's helemaal geen scripts, zelfs niet me
 
 Aan de andere kant van het spectrum laten de cijfers zien dat ongeveer 97% van de pagina's ten minste één script bevat, inline of extern.
 
-{# TODO(analysts): We still have a problem here with the x-axis label (“Containing”). Can someone help out and look at this? #}
-
 {{ figure_markup(
   image="script-use.png",
+  alt="Gebruik van het script element
```

Comment:
```suggestion
  alt="Gebruik van het script element,
```

Reply:
Ah yes I'll add those

Reply:
And it also needs the " after element right?